starboard/rdk/launcher.py

```diff
@@ -261,8 +269,14 @@ def _readloop():
         # Check for the test complete tag. It will be followed by either a
         # success or failure tag.
         if line.find(self.test_complete_tag) != -1:
-          self.return_value = 0
-          return
+          self.test_completed = 1
+        if self.test_completed:
+          self.test_completed += 1
+          if self.test_completed > 10:
```

Comment:
Why are you looking for 10 rows with the `test_complete_tag` before exiting?

Reply:
If the test fails, the test_failed_tag ("FAILED TEST") message will be printed in the following line of the the test_complete_tag. The WPEFrameworks log also comes with other framework information. The failed_tag not necessarily comes in the next line. Also in the case it passes, nothing will be printed to indicate that state. Therefore, I let it scan for the failed_tag for 10 lines to be safe before claiming pass.

Reply:
Waiting ten lines seems a little arbitrary. Does the test command complete on its own? Does it hang if you don't have this check?

Reply:
Without this 10 lines timeout, the test_runner will wait till 20-minute timeout (hardcoded elsewhere), and call it a fail with message like "has the test crashed?" or similar.

Reply:
This change is removed from this CL and will be revised.

Reply:
I will have a new CL for this part with revised changes.